sdk/dotnet/Pulumi.Automation/LocalWorkspace.cs

```diff
@@ -607,6 +607,31 @@ public override async Task<ImmutableList<StackSummary>> ListStacksAsync(Cancella
             return stacks.ToImmutableList();
         }
 
+        /// <inheritdoc/>
+        public override async Task<StackDeployment> ExportStackAsync(string stackName, CancellationToken cancellationToken = default)
+        {
+            var commandResult = await this.RunCommandAsync(
+                new [] { "stack", "export", "--stack", stackName, "--show-secrets" },
+                cancellationToken);
```

Comment:
For consistency with other methods in the class using await consider:

```suggestion
            var commandResult = await this.RunCommandAsync(
                new [] { "stack", "export", "--stack", stackName, "--show-secrets" },
                cancellationToken).ConfigureAwait(false);
```